core/controllers/acl_decorators.py

```diff
@@ -2506,6 +2506,17 @@ def test_can_accept_suggestion(
                     self.user_id, suggestion.score_category):
                 return handler(self, target_id, suggestion_id, **kwargs)
 
+            if suggestion.suggestion_type == (
```

Comment:
Perhaps. As soon as a reviewer leaves comments on your code, it means that there is at least one reader to whom the distinction is not clear. So I would suggest answering the question within the code itself, for the benefit of other readers of the code in the future. Adopt this as a general practice.

More specifically: you explained in a reply to the original comment that the condition will never be true and that the whole scoring structure will be removed in a future PR. Both of these points individually seem like important information that future readers of the code won't know anything about, and so it should be documented (unless I misinterpreted your comment above).

Reply:
> the condition will never be true and that the whole scoring structure will be removed in a future PR

Yeah, these conditions are True but I don't think a reader has to know this to understand the new changes OR the new changes are made considering "these" conditions i.e, it will work without considering ethe above condition.